modules/ocf_desktop/manifests/printnotify.pp

```diff
@@ -0,0 +1,27 @@
+class ocf_desktop::printnotify {
+  user { 'ocfbroker':
+    ensure => present,
+    shell  => '/bin/false',
+  }
+
+  # enable regular users to run notification script as ocfbroker
+  file { '/etc/sudoers.d/broker':
+    content => "ALL ALL=(ocfbroker) NOPASSWD: /opt/share/puppet/print-notify-handler\n\
+ALL ALL=(ocfbroker) NOPASSWD: /bin/kill\n",
```

Comment:
At this point I think just making a file for this would be better than putting it inline here.